internal/install/_static/terraform_deployer_run.sh

```diff
@@ -2,7 +2,7 @@
 
 set -euxo pipefail
 
-cp -r /stage/*.tf /workspace
+cp -r /stage/* /workspace
```

Comment:
nit: Please add a comment on why we want to extend the file range. You have a good justification.

Reply:
What do you think of:

```suggestion
# Terraform code may rely on content from other files than .tf files (es json, zip, html, text), so we copy all the content over
# See more: https://github.com/elastic/elastic-package/pull/603
cp -r /stage/* /workspace
```